EIPS/eip-7110.md

```diff
@@ -0,0 +1,213 @@
+---
+eip: 7110
+title: NFT Dynamic Ownership
+description: An innovative extension to NFT token that introduces dynamic ownership and nesting capabilities.
+author: hiddenintheworld.eth (@hiddenintheworld)
+discussions-to: https://ethereum-magicians.org/t/erc-7110-nft-dynamic-ownership/14516
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-01
+requires: 721
+---
+
+## Abstract
+
+A standard interface for non-fungible tokens (NFTs) with dynamic ownership, which extends the capabilities of the original `ERC-721` standard by enabling NFTs to be owned by either addresses or other NFTs. The proposed `ERC-721D` standard introduces dynamic ownership in the world of NFTs. Instead of a token being owned solely by an address, as it is in the `ERC-721` standard, tokens following the `ERC-721D` standard can be owned by either an address or another token. This opens up new possibilities and adds an extra layer of complexity and opportunity in the NFT space. This EIP outlines the rules and functions needed to support this dynamic ownership model while maintaining compatibility with `ERC-721` standards.
```

Comment:
When you're referring to another proposal, don't enclose it in backticks. You have to use a link the first time (per proposal), but after that you can just reference the proposal as normal.

When referring to your interfaces, however, you should use backticks.

```suggestion
A standard interface for non-fungible tokens (NFTs) with dynamic ownership, which extends the capabilities of the original [ERC-721](./eip-721.md) standard by enabling NFTs to be owned by either addresses or other NFTs. The proposed `ERC721D` interface introduces dynamic ownership in the world of NFTs. Instead of a token being owned solely by an address, as it is in the ERC-721 standard, tokens following the `ERC721D` interface can be owned by either an address or another token. This opens up new possibilities and adds an extra layer of complexity and opportunity in the NFT space. This EIP outlines the rules and functions needed to support this dynamic ownership model while maintaining compatibility with ERC-721 standards.
```